tests/test_aliases.py

```diff
@@ -55,7 +56,7 @@ class MyModel(BaseModel):
             model_config = ConfigDict(alias_generator=return_bytes)
             bar: Any
 
-    assert str(e.value) == "Config.alias_generator must return str, not <class 'bytes'>"
+    assert str(e.value) == IsStr(regex="alias_generator <function .*> must return str, not <class 'bytes'>")
```

Comment:
I purposely changed this so you would see what alias generator function was misbehaving. (If you define multiple for different models, I don't think you'd be able to tell from the traceback where it came from, and I didn't see any reason not to include it here.)